mystery-deckbuilder/Assets/Scripts/Card/CardBase.cs

```diff
@@ -0,0 +1,57 @@
+using System;
+using System.Collections.Generic;
+
+/*
+ * An abstract class that all conversation cards override/inherit from
+ * All of a cards atrributes and information will be stored in a dictionary which can only be accessed through getter methods.
+ * The values for the cards attributes are declared in derived card classes.
+ */
+
+public abstract class ConversationCard
+{
+
+    protected readonly int _id;
+    protected IDictionary<string, string> _metadata = new Dictionary<string, string>();
+    protected List<Action> methods = new List<Action>();
+
```

Comment:
NIT: "methods" is not a very descriptive name. I'm also unfamiliar enough with actions to not know if they are viable here, but that is subject to change! Just make sure you keep in contact with Will on how to use local effects